gitlab-logger.yaml

```diff
@@ -23,6 +23,10 @@ pipeline:
       output: gitlab-logger
       ldflags: -w
 
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/usr/local/bin
+      cp ${{targets.destdir}}/usr/bin/gitlab-logger "${{targets.destdir}}"/usr/local/bin/gitlab-logger
+
```

Comment:
I think we'll need a -compat subpackage for this, rather than using `/usr/local/` we can use a subpackage that creates a symlink to the binary `/usr/bin/gitlab-logger`  into `/usr/local/`

Reply:
i.e. like this https://github.com/wolfi-dev/os/blob/1c49ca89b30fb7f2c65553c1d4aaadd2d4417693/flux.yaml#L40-L47

the reason is in wolfi we avoid installing into /usr/local

Reply:
Awesome! Thanks for the explanation.